src/main/java/com/cascadebot/cascadebot/commands/core/ServerInfoCommand.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2018 CascadeBot. All rights reserved.
+ * Licensed under the MIT license.
+ */
+
+package com.cascadebot.cascadebot.commands.core;
+
+import com.cascadebot.cascadebot.CascadeBot;
+import com.cascadebot.cascadebot.commandmeta.CommandContext;
+import com.cascadebot.cascadebot.commandmeta.CommandType;
+import com.cascadebot.cascadebot.commandmeta.ICommand;
+import com.cascadebot.cascadebot.utils.DiscordUtils;
+import com.cascadebot.cascadebot.utils.pagination.Page;
+import com.cascadebot.cascadebot.utils.pagination.PageObjects;
+import net.dv8tion.jda.core.EmbedBuilder;
+import net.dv8tion.jda.core.entities.*;
+import org.bukkit.Server;
+
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class ServerInfoCommand implements ICommand {
+    @Override
+    public void onCommand(Member sender, CommandContext context) {
+
+        List<Page> pageList = new ArrayList<>();
```

Comment:
You don't need to use pages for this as their is only one page

Reply:
Oh yeah

Reply:
Because I don't know any other way to send the embed :p plz helP